test/ClangImporter/enum-cxx.swift

```diff
@@ -0,0 +1,12 @@
+// RUN: %target-swift-frontend -emit-sil %s -I %S/Inputs/custom-modules -module-cache-path %t -enable-cxx-interop -verify | %FileCheck %s
```

Comment:
Ah, I really did mean an IR-level test, so we can be sure that Swift lowers the enum to an i8-sized type rather than an uninhabited type.